pkg/cmd/openshift-sdn-node/kube_proxy.go

```diff
@@ -56,19 +54,20 @@ const (
 
 // This is a stripped-down copy of ProxyServer from
 // k8s.io/kubernetes/cmd/kube-proxy/app/server.go, and should be kept in sync with that.
+// (rr) definitely not in sync with https://github.com/openshift/kubernetes/blob/sdn-4.13-kubernetes-1.26.0/cmd/kube-proxy/app/server.go#L527
 type ProxyServer struct {
 	Client             clientset.Interface
 	IptInterface       utiliptables.Interface
 	execer             exec.Interface
 	Broadcaster        events.EventBroadcaster
 	Proxier            proxy.Provider
-	ProxyMode          string
+	ProxyMode          kubeproxyconfig.ProxyMode
 	NodeRef            *v1.ObjectReference
 	MetricsBindAddress string
 	EnableProfiling    bool
-	UseEndpointSlices  bool
 	ConfigSyncPeriod   time.Duration
 	HealthzServer      healthcheck.ProxierHealthUpdater
+	localDetectorMode  kubeproxyconfig.LocalMode
```

Comment:
This isn't actually overridable in openshift-sdn, so it's probably confusing to add this

Reply:
ok removed it